ivy.el

```diff
@@ -3210,7 +3239,16 @@ FACE is the face to apply to STR."
 (defun ivy--format-minibuffer-line (str)
   "Format line STR for use in minibuffer."
   (if (eq ivy-display-style 'fancy)
-      (funcall ivy--highlight-function (copy-sequence str))
+      (let ((str
+             (funcall ivy--highlight-function (copy-sequence str))))
+
+        (add-text-properties
+         0 (length str)
+         `(mouse-face
+           ivy-minibuffer-match-highlight
+           help-echo (format "mouse-1: %s" ivy-mouse-action-tooltip))
```

Comment:
Missing backquote comma?

Reply:
Indeed. It surprises me that it works without the comma...

Reply:
Actually it should come as no surprise - the [`help-echo` property can be evaluated as a function / form](https://www.gnu.org/software/emacs/manual/html_node/elisp/Special-Properties.html#index-help_002decho-_0040r_007b_0028text-property_0029_007d-3414), in which case I suggest reverting to deferred evaluation (i.e. replace backquote with plain quote), since `ivy-mouse-action-tooltip` is a variable.

Reply:
@mookid WDYT? (It just occurred to me that GitHub may not have notifed you of my replies on this and previous "outdated diffs". My apologies if you've already seen - I won't nag again.)